docs/tools/toolchains/export_to_third_party.md

```diff
@@ -6,16 +6,13 @@ If you'd like to develop on Arm Mbed OS with a third party tool, or migrate to o
 - IAR Systems.
 - Eclipse CDT ([C/C++ Development Tooling](https://www.eclipse.org/cdt/)) make (unmanaged) projects using:
     - GNU Arm Embedded Toolchain 6 (6-2017-q1-update).
-    - Arm Compiler 5.06 Update 6.
```

Comment:
Do we know if it works with Arm Compiler 6? If it does, we should replace with Arm Compiler 6. We should check with DSG. They should know straightaway.